File: programmers/난이도별/level01.자릿수_더하기/JiaLee0707.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int n) {
    int answer = 0;
    
    do {
        answer += n % 10;
        n /= 10;
    } while(n > 0);
    
    return answer;
}

File: programmers/난이도별/level01.[1차]_비밀지도/JiaLee0707.js
function solution(n, arr1, arr2) {
    var answer = [];
    
    arr1 = arr1.map((value) => value.toString(2));
    arr2 = arr2.map((value) => value.toString(2));
    
    for(let i=0; i<n; i++) {
        answer[i] = '';
        const value1 = arr1[i].padStart(n, '0');
        const value2 = arr2[i].padStart(n, '0');
        
        for(let j=0; j<n; j++) {
            if(value1[j] === '1' || value2[j] === '1') answer[i] += '#';
            else answer[i] += ' ';
        }
    }
    return answer;
}
